component/grade-table.js

```diff
@@ -10,48 +11,68 @@ class GradeTable {
         this.tableElement.innerHTML ="";
         //this.tableElement.empty();
 
+        this.renderGradeRow(grades, this.deleteGrade);

```

Comment:
The `.renderGradeRow` method should create a single table row (and its contents) and return it each time it's called, with the `.updateGrades` method appending each row to the table body.